### PR TITLE
Hidable support for series charts

### PR DESCRIPTION
**Pull request by mikwat**
In an effort to address #582, I've made some changes to "stack-mixin" to make it aware of series charts through the fact that the `_stack` has only one element. This is necessary because when series charts are redrawn, the `_stack` of each underlying chart is rebuilt. Therefore, any changes to the `hidden` attribute are not persisted.

I also updated "composite-chart" so that it filters hidden children in a few key locations.

I updated a few examples so that they demonstrate the hidable legends and added a stacked line (area) chart example.

If this seems like a reasonable solution, let me know and I'll add some specs. I've tried my best to follow the contribution guidelines, but this is my first time so please let me know if I've made any mistakes.

### CODE REVIEW

**Comment by mikwat** [comment]
@gordonwoodhull Any feedback appreciated, thanks.

**Comment by gordonwoodhull** [comment]
Sorry for the slow response, I am absolutely swamped. This looks really good at a first glance. It does need tests.

**Comment by mikwat** [comment]
Thanks, I understand. I'll add tests.

File: spec/series-chart-spec.js [hunk 1]
```diff
@@ -133,4 +133,50 @@ describe('dc.seriesChart', function() {
             expect(dots[0].length).toEqual(6);
         });
     });
+
+    describe('legends with hidable stacks', function () {
```
Comment (mikwat) [inline spec/series-chart-spec.js]: @gordonwoodhull Added specs. I copied these from `legend-spec.js` and modified slightly to work with `seriesChart`. Would it be better to consolidate these in `legend-spec.js`? If so, I'd need to duplicate the `seriesChart` setup. Not a big deal, just let me know which approach you prefer.

**Comment by mikwat** [comment]
I know you're busy @gordonwoodhull Feedback welcome whenever you have a chance.

**Comment by gordonwoodhull** [comment]
Thanks @mikwat. I tend to merge and fix any problems myself since I never know who will still be around to respond to code reviews.

This generally looks good and complete. I'll be sure to ping you on any revisions I make.

**Comment by mikwat** [comment]
Sounds good. I'll be around.

**Pull request closed without merging**